cri/Makefile

```diff
@@ -21,14 +21,14 @@
 # SOFTWARE.
 
 EXTRAGOARGS:=-v -race -cover
-CTRDLOGDIR:=/tmp/ctrd-logs
+CTRDLOGDIR:=/tmp/ctrd-logs/${GITHUB_RUN_ID}
```

Comment:
is this folder available on the host and where? please leave a comment

Reply:
why not ${{ github.run_id }}-${{ github.run_number }} here as well?

Reply:
${{ github.run_id }}-${{ github.run_number }} is stored in an evironment variable [here](https://github.com/ease-lab/vhive/blob/2b9439af058693532f55e6ecd5a28aca6c205ca1/.github/workflows/cri_test.yml#L55) which is then passed on to the makefile.